index.bs

```diff
@@ -1258,16 +1258,16 @@ the {{CredentialsContainer/create()}} or {{CredentialsContainer/get()}} methods.
 </pre>
 
 <div dfn-type="enum-value" dfn-for="Transport">
-    Authenticators may communicate with Clients using a variety of transports. This enumeration defines a hint as to how Clients
-    might communicate with a particular Authenticator in order to obtain an assertion for a specific credential. Note that these
-    hints represent the [=[RP]=]'s best belief as to how an Authenticator may be reached. A [=[RP]=] may obtain a list of
-    transports hints from some [=attestation statement formats=] or via some out-of-band mechanism; it is outside the scope of
+    Authenticators MAY communicate with Clients using a variety of transports. This enumeration defines a hint as to how Clients
```

Comment:
This is descriptive - not normative language. Please back this individual change out.

Reply:
done